client/js/components/menu.jsx

```diff
@@ -0,0 +1,26 @@
+var React = require("react");
+var $ = require("jquery");
+
+var Menu = React.createClass({
+  componentDidMount: function() {
+    $(document).ready(function() {
+      $(".menu").click(function () {
+        $("#dropdown").slideToggle(300);
```

Comment:
super minor, but the [magic number](http://dduugg.github.io/naming-and-defining-constants/) should be extracted.